Makefile

```diff
@@ -11,7 +11,7 @@ all: test
 NOVENDOR = go list github.com/tendermint/tmlibs/... | grep -v /vendor/
 
 test:
-	go test `glide novendor`
+	go test -tags gcc `glide novendor`
```

Comment:
Am not too sure if we should always be including the gcc tag, since on my Mac this is going to trip out.

Reply:
How come it trips on your Mac? It runs fine on macOS 10.13 for me.

Reply:
Works for me... go 1.8.1, 10.12.x

Reply:
Aha nice, my Mac has issues with gcc tags but I'll try again.

Reply:
This was where I first tripped out https://github.com/tendermint/merkleeyes/issues/39#issuecomment-322348864 2 months ago.